packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.jsx

```diff
@@ -50,23 +50,23 @@ const ContractDrawer = observer(
 
         const contract_audit = (
             <ContractAudit
+                contract_end_time={getEndTime(contract_info)}
                 contract_info={contract_info}
                 contract_update_history={contract_update_history}
-                contract_end_time={getEndTime(contract_info)}
+                duration_unit={getDurationUnitText(getDurationPeriod(contract_info))}
```

Comment:
@akmal-deriv nit: if it's only about alphabetical sorting of props, you might wanna revert this change :) but it's up to you

Reply:
This particular place looked messy to me, so I sorted it out. I think it's a better way as the code contains too many props here)